nav2_core/include/nav2_core/behavior_tree_navigator.hpp

```diff
@@ -371,6 +374,10 @@ class BehaviorTreeNavigator : public NavigatorBase
   rclcpp::Clock::SharedPtr clock_;
   FeedbackUtils feedback_utils_;
   NavigatorMuxer * plugin_muxer_;
+
+  // Error tracking
+  uint16_t current_error_code_;
```

Comment:
Would these be used then?

Reply:
See the comment from the previous review on getting the error codes from the BT actions and propogating through to the navigator like we do the error code

Reply:
keeping as discussed.